release(camera-plugin): 5.0.0 blog post (#1122)
apache/cordova-docs: www/_posts/2020-09-18-camera-plugin-release.md

---
layout: post
author:
    name: Bryan Ellis
title:  "Camera Plugin Released!"
categories: news
tags: release plugins
---

We are happy to announce that we have just released an update to our camera plugin!

* [cordova-plugin-camera@5.0.0](https://www.npmjs.org/package/cordova-plugin-camera)

## Release Highlights

In this major release, we have:

* removed the `DestinationType` of `NATIVE_URI` that was used only in the iOS and Windows platform. For iOS, the `NATIVE_URI` returned an `ALAssetsLibrary` that had been deprecated and does not work with the `WKWevView`.
* removed the `CordovaUri` class helper that used fuzzy logic to guess the image path from the content URL. This process was not necessary as we know the file path beforehand.
* bumped the project requirement definition. This plugin version requires Cordova CLI >= 9.0.0. If building for Android, this plugin version will also require the Cordova Android platform >= 9.0.0.

> * changed the location where temporary images are saved in. They are now saved in the cache directory. This allows the system to manage the cache cleanup and prevent using up all of the removable storage devices.

Reviewer: The change was from the external cache to the internal cache, so I don’t think this behavior changes that much.
According to this, it could happen that it doesn’t get deleted, but I understand that in general it deletes the files
https://developer.android.com/reference/android/content/Context#getExternalCacheDir()

> * changed the location where temporary images are saved in. They are now saved in the cache directory. This allows the system to manage the cache cleanup and prevent using up all of the removable storage devices.

Author: ```suggestion
* changed the location of the temporary saved images from external to internal cache.
```
